src/EnergyPlus/HeatingCoils.cc

```diff
@@ -2267,10 +2266,14 @@ namespace HeatingCoils {
             // some of these are reset in Init, can be removed to speed up code
             heatingCoil.ElecUseLoad = 0.0;
             heatingCoil.HeatingCoilLoad = 0.0;
-            ElecHeatingCoilPower = 0.0;
 
         } // end of on/off if - else
 
+        if (state.dataHeatingCoils->CoilIsSuppHeater) {
+            state.dataHVACGlobal->SuppHeatingCoilPower = heatingCoil.ElecUseLoad;
+        } else {
+            state.dataHVACGlobal->ElecHeatingCoilPower = heatingCoil.ElecUseLoad;
+        }
```

Comment:
These two variables are set in a single location depending the `state.dataHeatingCoils->CoilIsSuppHeater` flag variable.